package.json

```diff
@@ -28,9 +25,8 @@
   ],
   "license": "Apache-2.0",
   "scripts": {
-    "watch": "watchify src/main.js --standalone WebVRPolyfill -v -d -o 'derequire > build/webvr-polyfill.js'",
-    "build": "browserify src/main.js --standalone WebVRPolyfill | derequire > build/webvr-polyfill.js",
-    "min": "browserify src/main.js --standalone WebVRPolyfill | derequire | uglifyjs -c > build/webvr-polyfill.min.js",
+    "watch": "webpack --watch",
```

Comment:
can you change the `watch` script to call `webpack-dev-server` (have to also `npm install -S webpack-dev-server`)? or create a new command called `dev`, whichever you think. I think that running `npm start` should start the dev server.

this is what I changed the `package.json` to locally:

```diff
diff --git a/package.json b/package.json
index 9941968..b361896 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,8 @@
   "description": "Use WebVR today, on mobile or desktop, without requiring a special browser build.",
   "dependencies": {
     "eventemitter3": "^2.0.2",
-    "object-assign": "^4.0.1"
+    "object-assign": "^4.0.1",
+    "webpack-dev-server": "^2.4.5"
   },
   "devDependencies": {
     "chai": "^3.5.0",
@@ -25,7 +26,8 @@
   ],
   "license": "Apache-2.0",
   "scripts": {
-    "watch": "webpack --watch",
+    "start": "npm run dev",
+    "dev": "webpack-dev-server",
     "build": "webpack",
     "test": "mocha"
   },
```